homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/shop/Shop.java

```diff
@@ -0,0 +1,49 @@
+package com.tutrit.java.quickstart.homework.shop;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Shop {
+    Logger log = LoggerFactory.getLogger("Shop ");
+    Map<String, BigDecimal> myShop = new LinkedHashMap<>();
+    Product myProduct;
+    ProductBank bank;
+
+    @Override
+    public String toString() {
+        return " " +
+                "shop =" + myShop +
+                '}';
+    }
+
+    void addProduct(Product product) {
```

Comment:
make methods public

Reply:
Why?

Reply:
Best practice is evidently show access modifier for methods.

Reply:
See your methods below also.

Reply:
fixed